munich/README

```diff
@@ -1,7 +1,19 @@
 ###PyLadies Munich
```

Comment:
I would change this to `# PyLadies Munich`

Reply:
done :)